packages/KAlert/KAlert.vue

```diff
@@ -13,31 +15,71 @@
       {'isFixed': isFixed}
     ]"
     class="k-alert"
-    role="alert">
+    role="alert"
+    @click.stop>
     <button
-      v-if="isDismissible"
+      v-if="dismissType === 'icon'"
       type="button"
       aria-label="Close"
       class="close"
-      @click="dismissAlert()">
+      @click="dismissAlert">
       <KIcon
+        :color="appearance"
+        :class="appearance"
         icon="close"
-        color="#000"
-        size="12"
-        view-box="0 0 12 12" />
+        size="14" />
     </button>
+    <div
+      :class="appearance"
+      class="k-alert-action ml-3">
+      <!-- @slot Use this slot to pass extra buttons other than Dismiss  -->
+      <slot
+        v-if="hasActionButtons && (dismissType === 'button' || dismissType ==='none')"
```

Comment:
```suggestion
        v-if="hasActionButtons"
```

Let's just not constrain it on dismissType.

Reply:
Fixed!